addons/l10n_ar_withholding/views/res_partner_view.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record model="ir.ui.view" id="view_partner_form">
+        <field name="name">res.partner.form.inherit</field>
+        <field name="inherit_id" ref="account.view_partner_property_form"/>
+        <field name="model">res.partner</field>
+        <field name="arch" type="xml">
+            <group name="electronic_invoicing" position="after">
+                <group string="Withholding (AR)" invisible="'AR' not in fiscal_country_codes">
+                    <field name="l10n_ar_partner_tax_ids" nolabel="1" colspan="2" widget="auto_save_res_partner">
+                        <tree editable="top">
+                            <field name="tax_id" options="{'no_create': True, 'no_open': True}" domain="[('l10n_ar_withholding_payment_type', '!=', False)]"/>
```

Comment:
```suggestion
                            <field name="tax_id" options="{'no_create': True, 'no_open': True}" domain="[('l10n_ar_withholding_payment_type', '=', 'supplier')]"/>
```

Reply:
@vbe-odoo on res.partner form view I think we should be able add taxes for vendor or supplier payments

Reply:
@pablohmontenegro make the change